site/content/docs/tutorials/workflow-orchestration/multikueue-dws/index.md

```diff
@@ -0,0 +1,102 @@
+---
+linkTitle: "Multikueue, DWS and GKE Autopilot"
+title: "Multikueue, DWS and GKE Autopilot"
+description: "In this guide you will learn how to set up a multi-cluster environment where job computation is distributed across three GKE clusters in different regions using MultiKueue, Dynamic Workload Scheduler (DWS), and GKE Autopilot."
+weight: 30
+type: docs
+owner: >-
+    [Jean-Baptiste Leroy](https://github.com/leroyjb)
+tags:
+ - Orchestration
+ - Tutorials
+ - Kueue
+ - DWS
+draft: false
+---
+
+This repository provides the files needed to demonstrate how to use [MultiKueue](https://kueue.sigs.k8s.io/docs/concepts/multikueue/) with [Dynamic Workload Scheduler](https://cloud.google.com/blog/products/compute/introducing-dynamic-workload-scheduler?e=48754805) (DWS) and [GKE Autopilot](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-overview).  This setup allows you to run workloads across multiple GKE clusters in different regions, automatically leveraging available GPU resources thanks to DWS.
```

Comment:
Remember that this will be now part of the site, so you need to tell the user which repository they need to clone and adapt a bit the wording